week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,12 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  return price * 1.4;
+}
+function convertToBRL(price) {
+  return price * 5 + price * 5 / 100;
```

Comment:
The fee should be added for conversion to any foreign currency, so it should be done for converToUSD as well. Can you think of sharing the fee code?